README.md

```diff
@@ -109,14 +109,13 @@ Then, install the  Kunpeng Accelerator Engine:
 
 ```
 cd KAE 
-chmod +x configure 
```

Comment:
I've changed the exec permissions of this file and there is no need of this step anymore.

Reply:
Cool, it the good idea to update it. and I also see the configure changes from 644 to 755.

Note for other reviewers, it can be update using cmd `git update-index --chmod=+x  filename` to add the exec permissions.